Xamarin.Forms.Platform.Android/Renderers/PageRenderer.cs

```diff
@@ -117,7 +120,9 @@ void UpdateBackground(bool setBkndColorEvenWhenItsDefault)
 				bool isDefaultBkgndColor = bkgndColor.IsDefault;
 				if (page.Parent is BaseShellItem && isDefaultBkgndColor)
 				{
-					var color = Context.Resources.GetColor(global::Android.Resource.Color.BackgroundLight, Context.Theme);
+					var color = Forms.IsMarshmallowOrNewer ?
+						Context.Resources.GetColor(AColorRes.BackgroundLight, Context.Theme) :
+						new AColor(ContextCompat.GetColor(Context, global::Android.Resource.Color.BackgroundLight));
```

Comment:
Can we do this one, too?
```suggestion
						new AColor(ContextCompat.GetColor(Context, AColorRes.BackgroundLight));
```